Networking/Networking/Remote/ProductsRemote.swift

```diff
@@ -478,6 +489,40 @@ public final class ProductsRemote: Remote, ProductsRemoteProtocol {
         let mapper = ProductReportListMapper()
         return try await enqueue(request, mapper: mapper)
     }
+
+    public func loadVariationReports(for siteID: Int64,
+                                     productIDs: [Int64],
+                                     variationIDs: [Int64],
+                                     timeZone: TimeZone,
+                                     earliestDateToInclude: Date,
+                                     latestDateToInclude: Date,
+                                     pageSize: Int,
+                                     pageNumber: Int,
+                                     orderBy: ProductsRemote.OrderKey,
+                                     order: ProductsRemote.Order) async throws -> [ProductReport] {
+        let dateFormatter = DateFormatter.Defaults.iso8601WithoutTimeZone
+        dateFormatter.timeZone = timeZone
+        let path = Path.variationReports
+        let parameters: [String: Any] = [
+            ParameterKey.products: productIDs,
+            ParameterKey.variations: variationIDs,
+            ParameterKey.before: dateFormatter.string(from: earliestDateToInclude),
+            ParameterKey.after: dateFormatter.string(from: latestDateToInclude),
```

Comment:
nit: I think `earliestDateToInclude` should be sent as `after` and `latestDateToInclude` as `before`. I left a similar comment [here](https://github.com/woocommerce/woocommerce-ios/pull/12803#discussion_r1615438165)

Reply:
Great catch! Updated for both the product and variation report requests in 11cc064dc5f98ae3bd379966794a424d6c0f654b.